__tests__/app.test.js

```diff
@@ -82,3 +82,64 @@ describe("GET /api/technicians/:user_id", () => {
       });
   });
 });
+
+describe('PATCH /api/technicians/:user_id', () => {
+  const patchData = {
+      services: [
+        "Servicing and MOT",
+        "Clutch repairs",
+        "Engine and cooling",
+        "Tyre Replacement",
+        "General Servicing"
+      ]
+  }
+  test('should accept an object of services, update the technician, and return it', () => {
+  const newTechnician = {
```

Comment:
This is a bit overkill, since we're only changing part of the object by id we only need to check the id and parts we updated, however maybe get other peoples input on this as I'm not known for thoroughly testing lol